test/tests/functional/pbs_two_mom_hooks_resources_used.py

```diff
@@ -103,6 +104,8 @@ def test_Rrecord(self):
         # Wait for the job to start running.
         self.server.expect(JOB, {ATTR_substate: '42'}, jid1)
```

Comment:
I'd like to remain consistent with other tests.  I've seen tests check for state: 'R' or both state and substate.  The latter being during a period before we fixed expect to check for substate itself.  It's not needed any more.  I've not seen tests just check for substate before.